tests/integration/nodes/writers/test_qdrant_writer_flow.py

```diff
@@ -90,6 +90,5 @@ def test_write_workflow(
     )
     assert response.output == expected_output
 
-    mock_set_up_collection.assert_called_once()
```

Comment:
why did you remove this check?

Reply:
There was an error with setting up the collection while rewriting the documents.
https://github.com/dynamiq-ai/dynamiq/pull/21/files/27826cf68cb6b5485e37b5df1affe52235f36a68#diff-f0eb78186b536379a2b20893abe3fe3f2460976ece7e1f525afeff0c466809e9L337

Reply:
So if we disabled rewriting the documents and this mock is not called let's change it to 
`mock_set_up_collection.assert_not_called()`